infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/StatusLogger.java

```diff
@@ -383,6 +386,30 @@ public void posActivated(final String pandas) {
     log.info(pandas);
   }
 
+  public void loadingDepositSnapshot(final String snapshotResource) {
+    log.info("Loading deposit tree snapshot from {}", snapshotResource);
+  }
+
+  public void loadedDepositSnapshotResource(
+      final List<Bytes32> finalized, final long deposits, final Bytes32 executionBlockHash) {
+    log.info(
+        "Loaded deposits tree state from snapshot with {} deposits and {} execution block hash. Finalized: {}.",
```

Comment:
I don't think we need to load the finalized roots - they won't mean anything to users (or developers generally).